modules/widgets/eu-cookie-law.php

```diff
@@ -99,16 +99,19 @@ function __construct() {
 		 */
 		function enqueue_frontend_scripts() {
 			wp_enqueue_style( 'eu-cookie-law-style', plugins_url( 'eu-cookie-law/style.css', __FILE__ ), array(), '20170403' );
```

Comment:
Since we're editing the contents of that file, let's bust the cache:

```suggestion
			wp_enqueue_style( 'eu-cookie-law-style', plugins_url( 'eu-cookie-law/style.css', __FILE__ ), array(), JETPACK__VERSION );
```

Reply:
@jeherve This has been fixed in the latest commit.